pkgs/servers/monitoring/uptime-kuma/default.nix

```diff
@@ -0,0 +1,55 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, fetchurl, substituteAll, iputils }:
+let
+  deps = import ./composition.nix { inherit pkgs; };
+
+  dist = stdenv.mkDerivation rec {
```

Comment:
Add this as a passthru into the main derivation, use `mkDerivation (finalAttrs: { ... })` rather than `rec` so we can override `dist`

```
passthru.ui = dist;
```

Reply:
Done !